lexos/processors/prepare/scrubber.py

```diff
@@ -787,11 +642,98 @@ def load_cached_file_string(cache_folder: str, filename: str) -> str:
         return ""
```

Comment:
also, technically, those are not really caches (caches stores information to speeds up computation). Can you think of a better names for them?